File: PROGRAMMERS/lv0/짝수 홀수 개수.js
function solution(num_list) {
  let ans = [0, 0];

  for (let i of num_list) {
    ans[i % 2]++;
  }

  return ans;
}

let inps = [
  [1, 2, 3, 4, 5],
  [1, 3, 5, 7],
];

for (let i of inps) {
  console.log(solution(i));
}

File: PROGRAMMERS/lv0/python/짝수 홀수 개수.py
def solution(num_list):
    ans = [0, 0]

    for num in num_list:
        ans[num%2] += 1

    return ans

inps = [
    [1, 2, 3, 4, 5],
    [1, 3, 5, 7]
]

for inp in inps:
    print(solution(inp))